packages/dashboard-core-plugins/src/panels/ChartPanel.test.tsx

```diff
@@ -72,6 +72,7 @@ function makeChartPanelWrapper({
   glContainer = makeGlComponent(),
   glEventHub = makeGlComponent(),
   columnSelectionValidator = undefined,
+  makeApi = () => Promise.resolve(dh),
```

Comment:
This doesn't match `ChartPlugin`, which just returns `dh` and not a Promise...

Reply:
Updated `ChartPlugin` to return `Promise.resolve(dh)`.